services/core-api/tests/factories.py

```diff
@@ -354,6 +355,22 @@ def documents(obj, create, extracted, **kwargs):
             size=extracted, incident=obj, mine_document__mine=None, **kwargs)
 
 
+class MineIncidentNoteFactory(BaseFactory):
```

Comment:
Do you want some fake notes generated when you mock mine data cause if you would need to add an after generate to the incident to add some.